modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,39 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved, by specifying a concurrent number of VSBs and VSRs simultaneously in Progress.
+
+After all async plugin operations are completed, the backup is marked as completed.
+The Data Mover solution uses the Restic option of VolSync.
+
 
 [NOTE]
 =====
 Data Mover supports backup and restore of CSI volume snapshots only.
-
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
 =====
 
 .Prerequisites
 
-* You have verified that the `StorageClass` and `VolumeSnapshotClass` custom resources (CRs) support CSI.
+* You have verified that the `StorageClass` and `VolumeSnapshotClass` Custom Resources (CRs) support CSI.
 
 * You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
 
+* You have verified that `volumeSnapshotClass` CR `deletionPolicy` is set to `Retain`.
+
 * You have verified that only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
 
 * You have included the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR.
 
+* You have verified that the `OADP namespace` has the annotation `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`.
```

Comment:
This step should not be required for OADP 1.2 and volsync 0.7.z @eemcmullan  correct?

Reply:
@weshayutin I think it can still be used, as well as the new volumeOptions configs: https://volsync.readthedocs.io/en/stable/usage/permissionmodel.html

Reply:
@CarmiWisemon so we should note something like ...
In OADP 1.1.x  the following setting was mandatory
```
* You have verified that the `OADP namespace` has the annotation `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`.
```
In OADP 1.2.x  using the privileged-movers setting in most cases is not required.  The restoring container permissions should be adequate for the volsync copy.  In some user scenarios permission errors could be encountered where using the priviledged-mover='true' setting would resolve the issue.

Reply:
@weshayutin I implemented your comments and Tiger's comments, and I'm sending this PR for our internal peer-review by @RichardHoch

Reply:
oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`. is NOW done in code and is no longer a required doc / human step in 1.2.0